docs/ref/request-response.txt

```diff
@@ -290,6 +290,9 @@ Methods
 
     Example: ``"127.0.0.1:8000"``
 
+    Raises a ``django.core.exceptions.DisallowedHost`` exception if the host
+    is invalid.
```

Comment:
I would explicitly mention "... and not in [`ALLOWED_HOSTS`](https://docs.djangoproject.com/en/3.1/ref/settings/#allowed-hosts)" so people understand this isn't just about whether it parses correctly. See here for the code / error messages that `get_host()` provides: https://github.com/django/django/blob/41e6b2a3c5e723256506b9ff49437d52a1f3bf43/django/http/request.py#L130-L135

Reply:
@cjerdonek I've made the changes.